packages/datatrak-web/src/components/CodeGeneratorQuestion/CodeGeneratorQuestion.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import { TextField, Typography } from '@material-ui/core';
+import React, { useEffect } from 'react';
+import styled from 'styled-components';
+import { SHORT_ID, generateMongoId, generateShortId } from './generateId';
+import { SurveyQuestionInputProps } from '../../types';
+
+const LabelId = styled(Typography).attrs({
+  variant: 'h1',
```

Comment:
This shouldn't be an `h1` - there should only ever be one `h1` on a page, and it will always be the page title. I think this should be a `label` element or an `h3`